src/pages/Home/PriceTable.tsx

```diff
@@ -0,0 +1,68 @@
+import { Button, Heading, Stack } from '@chakra-ui/react'
+import type { Asset } from '@shapeshiftoss/types'
+import { useCallback, useMemo } from 'react'
+import { FaStar } from 'react-icons/fa6'
+import { useTranslate } from 'react-polyglot'
+import { useHistory } from 'react-router'
+import type { Row } from 'react-table'
+import { MarketsTable } from 'components/MarketsTable'
+import { ResultsEmpty } from 'components/ResultsEmpty'
+import { Text } from 'components/Text'
+import { selectAssetsSortedByMarketCap, selectWatchedAssets } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+const starFilled = <FaStar />
+const emptyButtonProps = { size: 'lg', width: 'full' }
+
+export const PriceTable = () => {
+  const watchedAssetIds = useAppSelector(selectWatchedAssets)
+  const assets = useAppSelector(selectAssetsSortedByMarketCap)
+  const history = useHistory()
+  const translate = useTranslate()
+  const rows = useMemo(() => {
+    return assets.filter(asset => watchedAssetIds.includes(asset.assetId))
+  }, [assets, watchedAssetIds])
+
+  const handleButtonClick = useCallback(() => {
+    history.push('/assets')
+  }, [history])
+
+  const handleRowClick = useCallback(
+    (row: Row<Asset>) => {
+      const { assetId } = row.original
+      const url = assetId ? `/assets/${assetId}` : ''
+      history.push(url)
+    },
+    [history],
+  )
+  const body = useMemo(() => {
+    if (watchedAssetIds.length) {
+      return (
+        <>
+          <MarketsTable rows={rows} onRowClick={handleRowClick} />
+          <Button mx={6} onClick={handleButtonClick}>
+            {translate('watchlist.empty.cta')}
+          </Button>
+        </>
+      )
+    }
+    return (
```

Comment:
@reallybeard not sure about making this the default view - that's the default view for "Home" - if the user doesn't have assets in watchlist yet, then they'll simply be greeted with an empty state screen. While this is the intent of this PR, I'm not sure this is the best experience:
- Many users may never want to use a watchlist
- Even if they do, they may only like it as a secondary* view

e.g in Coinmarketcap, "Watchlist" is a secondary view which you can click to access, but the default view is assets

<img width="1481" alt="image" src="https://github.com/shapeshift/web/assets/17035424/5f23d590-ecf0-4118-8eed-8f962dfe2a00">

Reply:
Also since we're not just a CMC-like but also a wallet / dApp, what about including the balances in addition to prices, a la Zapper/Zerion?
In other words, this isn't only a watchlist for prices, but also for the balance of your starred assets

Reply:
The idea for Home is going to be a section for us to highlight things and it will grow in time as we add more stuff. 

All the balance stuff is in the wallet page.